src/components/annotations3d/attributes.js

```diff
@@ -75,13 +75,13 @@ module.exports = {
     arrowwidth: annAtts.arrowwidth,
     standoff: annAtts.standoff,
     hovertext: annAtts.hovertext,
-    hoverlabel: annAtts.hoverlabel
+    hoverlabel: annAtts.hoverlabel,
+    captureevents: annAtts.captureevents
```

Comment:
> would you prefer leaving subplotId out of the cartesian/paper-ref annotation event data or keep it as subplotId: false

haha didn't see this before [my comment](https://github.com/plotly/plotly.js/pull/1638#commitcomment-22295724)

yeah, lets omit it.